src/types.ts

```diff
@@ -44,3 +45,123 @@ export interface IssuerMetadata {
   jwks?: JSONWebKeySet;
   jwks_uri?: string;
 }
+
+export type HasherConfig = {
+  algo: string;
+  callback: Hasher;
+};
+
+export type SignerConfig = {
+  algo: supportedAlgorithm;
+  callback: Signer;
+};
+
+/**
+ * JOSE Types
+ */
+
+/** Recognized JWT Claims Set members, any other members may also be present. */
+export interface JWTPayload {
```

Comment:
@vijayshiyani I believe some of these types will be added to `sd-jwt` library and some of these definitions will be removed?

Reply:
@vijayshiyani I've added the following types
`JWK`
`JWTHeaderParameters`
`JWTPayload`

you can import these types from the latest `sd-jwt` lib

Reply:
I have updated above types to be  used from `sd-jwt`